workshops/book-dash/book-dash-mcr-report.md

```diff
@@ -0,0 +1,162 @@
+# The Turing Way | Book Dash - Manchester
+
+
+* Intro to Book Dash MCR slides: https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/book-dash/IntroBookDashMCR.pdf
+* HackMD: http://bit.ly/book-dash-mcr
+
+## The participants
+
+![https://pbs.twimg.com/media/D6y7be5XYAAF0i2?format=jpg&name=900x900](https://pbs.twimg.com/media/D6y7be5XYAAF0i2?format=jpg&name=900x900)
```

Comment:
Fun! Jez might have a better version of this picture as he had a nicer camera?

Can we make a caption that identifies the participants (back row, left to right....., middle row, left to right etc)? Will be really useful when reviewing in the future!